src/methods.ts

```diff
@@ -329,8 +329,19 @@ export type MPIMRepliesArguments = TokenOverridable & {};
   /*
    * `oauth.*`
    */
-export type OAuthAccessArguments = TokenOverridable & {};
-export type OAuthTokenArguments = TokenOverridable & {};
+export type OAuthAccessArguments = {
+  client_id: string;
+  client_secret: string;
+  code: string;
+  redirect_uri?: string;
+};
+export type OAuthTokenArguments = {
+  client_id: string;
+  client_secret: string;
+  code: string;
+  redirect_uri?: string;
+  single_channel?: boolean;
```

Comment:
i'm not 100% sure if this is a boolean, or if it should be `'0' | '1'`. waiting for confirmation.

Reply:
it does not support boolean.

> Currently it’s implemented to support specifically `1`.  Anything else will resolve to boolean `false` behind the scenes.